client/components/shared/Input.tsx

```diff
@@ -45,19 +40,20 @@ export const Input = (props: InputProps) => {
 		<label
 			css={css`
 				display: block;
-				color: ${neutral['7']};
+				color: ${palette.neutral['7']};
 				${textSans.medium()};
 				font-weight: bold;
 				${props.additionalCss}
 			`}
+			data-qm-masking="blocklist"
```

Comment:
so this would mask any input? I'm curious about this component, I wonder if it could be replaced by a source component, but the props seem quite different

Reply:
Yes any input although I'll be interested what QM captures. There is definitely some areas that could do with some refactoring.